lib/spanner_client_ext.rb

```diff
@@ -35,6 +35,31 @@ def commit_transaction transaction, mutations = []
           Convert.timestamp_to_time resp.commit_timestamp
         end
 
+        # Create a single-use transaction selector.
+        def single_use_transaction opts
+          return nil if opts.nil? || opts.empty?
+
+          exact_timestamp = Convert.time_to_timestamp \
+            opts[:timestamp] || opts[:read_timestamp]
```

Comment:
why are there two different names here? `:timestamp` VS `:read_timestamp`.

Reply:
Good point. The client library accepts both `:timestamp` and `:read_timestamp`, so I did the exact same thing here. But the code that is parsing the hints only accept `read_timestamp` (and will also only use `:read_timestamp` as an option), so this is not needed. The question is rather:
1. Do we want to support only the hint `read_timestamp: <value>`?
2. Or do we want to also support the hint `timestamp: <value>` in ActiveRecord?

WDYT?

Reply:
I would go for (1) to avoid the confusion.

Reply:
Done